src/userbase-js/src/db.js

```diff
@@ -723,6 +726,7 @@ const insertItem = async (params) => {
       case 'DatabaseIdCannotBeBlank':
       case 'DatabaseIdInvalidLength':
       case 'DatabaseIdNotAllowed':
+      case 'DatabaseIdNotAllowedForOwnDatabase':
```

Comment:
Don't need here or in updateItem, deleteItem, putTransaction, uploadFile, or getFile (will throw DatabaseNotOpen error bc can't open the database)

Reply:
All done